### PR TITLE
Implement PERMITTEDLIST_IDS env var

### PR DESCRIPTION
**Pull request by vanastassiou**
Since the base game config supports a `permittedlist.txt`, it makes sense to me to implement support for a `PERMITTEDLIST_IDS` environment variable (similarly to how `ADMINLIST_IDS` and `BANNEDLIST_IDS` have been implemented).

Test plan (testing pending):

1. - [ ] Install dev branch per [README](https://github.com/lloesche/valheim-server-docker#basic-docker-usage)
2. - [ ] Set `PERMITTEDLIST_IDS` variable in `valheim.env`
3. - [ ] Enter `This is a test string and should be deleted if my code works` in `/home/valheim-server/config/permittedlist.txt` 
4. - [ ] Start Valheim server
5. - [ ] Check `permittedlist.txt` to ensure the string in Step 3 does not exist and the expected list of Steam IDs exists
6. - [ ] Confirm whitelisted Steam user **can** log into server
      - [ ] Admin user
      - [ ] Non-admin user
7. - [ ] Confirm non-whitelisted Steam user (non-admin) **cannot** log into server
       - [ ] Admin user
      - [ ] Non-admin user

### CODE REVIEW

**Comment by lloesche** [comment]
@vanastassiou thank you for this PR! Can you please rebase against the current master and add the missing newline in `defaults`? Then we should be good to merge.

File: defaults [hunk 1]
```diff
@@ -128,6 +128,7 @@ POST_SERVER_SHUTDOWN_HOOK=${POST_SERVER_SHUTDOWN_HOOK:-}
 PRE_BEPINEX_CONFIG_HOOK=${PRE_BEPINEX_CONFIG_HOOK:-}
 POST_BEPINEX_CONFIG_HOOK=${POST_BEPINEX_CONFIG_HOOK:-}
 
-# Adminlist/Bannedlist IDs
+# Adminlist/Bannedlist/Permittedlist IDs
 ADMINLIST_IDS=${ADMINLIST_IDS:-}
 BANNEDLIST_IDS=${BANNEDLIST_IDS:-}
+PERMITTEDLIST_IDS=${PERMITTEDLIST_IDS:-}
```
Comment (lloesche) [inline defaults:134]: newline missing

**Review by lloesche — CHANGES_REQUESTED** [review]

File: defaults (hunk 1 above)
Comment (vanastassiou) [reply defaults]: Added newline

**Comment by vanastassiou** [comment]
@lloesche Sorry I wasn't clear about the test plan -- those were the steps I planned to have my friend running the server try out when he had time. I didn't want to put that work on you! If you think this PR is safe enough to merge without going through those steps, I'm happy to update the original comment to that effect and mark this ready for review.

**Review by lloesche — APPROVED** [review]

**Comment by lloesche** [comment]
@vanastassiou seems pretty low risk to me as we're not changing existing behavior. Code looks good to me 👍 🚢

**Pull request merged**